files/default/neutron-ha-tool.py

```diff
@@ -106,12 +106,12 @@ def parse_args():
                          'some reason.')
     wait_parser = ap.add_mutually_exclusive_group(required=False)
     wait_parser.add_argument('--wait-for-router', action='store_true',
-                    dest='wait_for_router')
+                             dest='wait_for_router')
```

Comment:
You are right, they should have been in separate commits - Let me split them - although to be fair, the commit message said that it contains both flake8 and unit testing.